agent/stats/queue_test.go

```diff
@@ -238,7 +239,7 @@ func TestQueueAddRemove(t *testing.T) {
 		t.Error("Sum value incorrectly set: ", *storageReadStatsSet.Sum)
 	}
 
-	storageWriteStatsSet, err := queue.GetStorageWriteStatsSet()
+	storageWriteStatsSet := storageStatsSet.WriteSizeBytes
```

Comment:
why is err ignored here? also below error message should be `Error getting storage write stats set`

Reply:
actually the `err` is logged only once, above on line 222 where we get `storageStatsSet`.  I'm removing this unnecessary err condition altogether.